### DISCUSSION TITLE
Is there a way to convert a header line into text line?

### DISCUSSION THREAD

**Discussion by quank123wip** [Milkdown/milkdown#338] [General]
It seems like a very basic feature and marktext have it. I don't know how to do it in milkdown.

**Comment by quank123wip** [comment]
Seems like the online demo's menu doesn't have this feature... Is there a hidden hotkey to do this?

**Comment by quank123wip** [comment]
Although the menu plugin is pretty configurable, I think this feature should be included in the example...

**Comment by quank123wip** [comment]
I made a quick insight into the code base... I think there isn't such a feature, need to add a "TurnIntoText" command, and add a option in the example config. But the solution is dirty and not neat for general use, such as convert code blocks into raw texts. Need more information for this poor idea...

**Comment by Saul-Mirone** [comment]
https://milkdown.dev/keyboard-shortcuts#paragraph

**Reply by quank123wip** [reply]
Oh, forgive me missed the documentations😥Sorry for bothering you for such a simple problem. But would it be better to include this feature in the example menu config? It should be more friendly for more users having a quick dive into the editor🥰If you think this is neat, I will make a PR soon.

**Reply by Saul-Mirone** [reply]
Sure, PR welcome.
